src/styles/blocks/about.scss

```diff
@@ -0,0 +1,37 @@
+.about {
+  &__title {
+    margin: 0 0 48px;
```

Comment:
there is no need to reset all your margins here to 0 because u did it for the whole page =)

Reply:
you can use margin-bottom, this way is more readable

Reply:
nope) it's h1 here. it has default margins

Reply:
oh really,  you're right. then you can reset them in one place, it will make your code more readable =)

or just use [reset](https://cdnjs.com/libraries/meyer-reset) library